tests/python_client/testcases/test_index.py

```diff
@@ -442,8 +443,7 @@ def test_create_index_on_field(self, connect, collection, get_simple_index):
         """
         tmp_field_name = "int64"
         result = connect.insert(collection, default_entities)
-        with pytest.raises(Exception) as e:
-            connect.create_index(collection, tmp_field_name, get_simple_index)
+        connect.create_index(collection, tmp_field_name, get_simple_index)
```

Comment:
this case need a re-design, I think. @binbinlv

Reply:
@yanliang567 OK, I will rewrite these scenario after this PR merged.